test/unit/Services/HttpServiceTest.php

```diff
@@ -169,6 +169,38 @@ static function ($headers) use ($locale) {
         $httpServiceMock->send('/my/uri/123', $resource);
     }
 
+    /**
+     * Verify 'CLIENTIP' header only set when a clientIp is defined in the Unzer object.
+     *
+     * @test
+     * @dataProvider clientIPHeaderShouldOnlyBeSetIfPropertyIsNotEmptyDP
+     *
+     * @param $clientIp
+     */
+    public function clientIPHeaderShouldOnlyBeSetIfPropertyIsNotEmpty($clientIp): void
```

Comment:
the header should not exist if the ip is not set.

Reply:
Not sure what you exactly mean with it but i have updated the name to make it better fitting.